addons/html_builder/static/src/snippets/snippet_service.js

```diff
@@ -430,6 +430,27 @@ export class SnippetModel extends Reactive {
             );
         });
     }
+
+    /**
+     * Gets the label of the snippet.
+     *
+     * @param {HTMLElement} snippetEl
+     * @returns {String}
+     */
+    getSnippetLabel(snippetEl) {
+        const label = snippetEl.dataset.oLabel;
+        // Check if any element in the snippet has the "parallax" class to show
+        // the "Parallax" label. This must be done this way because a theme or
+        // custom snippet may add or remove parallax elements. Note that if a
+        // label is already set, we do not change it.
+        let hasParallax = false;
+        if (!label) {
+            const contentEl = snippetEl.children[0];
+            hasParallax = contentEl.matches(".parallax")
```

Comment:
In fact, the parallax notion (and its class) is a website concept. This should not be in `html_builder`.